src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -4879,18 +3299,21 @@ void EvaluateIntHcModels(EnergyPlusData &state,
     // now call appropriate function to calculate Hc
     switch (ConvModelEquationNum) {
 
-    case ConvectionConstants::HcInt_UserCurve:
-        CalcUserDefinedInsideHcModel(state, SurfNum, state.dataSurface->SurfIntConvHcUserCurveIndex(SurfNum), tmpHc);
-        break;
-    case ConvectionConstants::HcInt_ASHRAEVerticalWall:
+    case HcInt::UserCurve: {
+        tmpHc = CalcUserDefinedIntHcModel(state, SurfNum, state.dataSurface->SurfIntConvHcUserCurveNum(SurfNum));
```

Comment:
Moved `Hc` to return value for several functions (it was already that way for most of them).